packages/babel-plugin-ember-test-metadata/__tests__/utils-test.js

```diff
@@ -32,111 +30,112 @@ describe('getNodeProperty', () => {
 });
 
 describe('getNormalizedFilePath', () => {
-  const fileOpts = {
-    embroiderBuildPath: {
-      root: '',
-      filename: path.join(
+  const appName = "example-app";
+  const moduleName = "example-app";
+  const addonName = "example-addon";
+  const addonPath = path.join('lib', addonName);
+  const config = {
+    pkg: {
+      name: appName
+    },
+  }
+
+  describe("given file path from a classic build", () => {
+    it("returns the normalized filepath", () => {
+      const expectedPath = path.join('tests', 'acceptance', 'example-test.js');
+      const preFormattedPath = path.resolve(appName, moduleName, expectedPath);
```

Comment:
I don't think we want `path.resolve` here. I think we should try to mirror, exactly, the values we identified, and each test should test multiple variations of paths.

We should just add each of these in our tests.

```
/Users/tester/workspace/personal/test-bed/classic/classic/tests/acceptance/foo-test.js
/Users/tester/workspace/personal/test-bed/classic/classic/tests/acceptance/subdir/foo-test.js
/Users/tester/workspace/personal/test-bed/classic/classic/tests/acceptance/subdir/subdir/foo-test.js
```

I've also thought about whether we need to use `path.join` or not, and I don't think we do. We don't need to support windows.